common/src/main/java/com/wynntils/models/beacons/BeaconModel.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.beacons;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.components.Model;
+import com.wynntils.mc.event.AddEntityEvent;
+import com.wynntils.mc.event.RemoveEntitiesEvent;
+import com.wynntils.mc.event.TeleportEntityEvent;
+import com.wynntils.models.beacons.event.BeaconEvent;
+import com.wynntils.models.beacons.type.BeaconColor;
+import com.wynntils.models.beacons.type.UnverifiedBeacon;
+import com.wynntils.models.beacons.type.VerifiedBeacon;
+import com.wynntils.utils.type.TimedSet;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import net.minecraft.core.Position;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.EntityType;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class BeaconModel extends Model {
+    // Amount of armor stands above each other to consider this a beacon
+    private static final int VERIFICATION_ENTITY_COUNT = 6;
```

Comment:
Is this a good margin? How many are there typically? 7 or 27?

Reply:
There are like 128/7.5 of them. This is more than enough.